src/Components/Header.jsx

```diff
@@ -63,6 +63,13 @@ export const Header = () => {
             >
               Getting Help
             </Link>
+            <Link
+              className="navbar-item"
+              to="/resources"
+              onClick={handleNavigate}
+            >
+              Resources
+            </Link>
```

Comment:
Not a blocking issue on this PR, but I think we would discuss whether we want to keep adding all the pages onto the navbar, as there's starting to become quite a few.

As an aside, it doesn't look like there's been a tile added to the homepage for this page yet 😄